picard/file.py

```diff
@@ -682,10 +702,12 @@ def state(self, state):
             return
         if state == File.PENDING:
             File.num_pending_files += 1
-            self.tagger.tagger_stats_changed.emit()
+            if hasattr(self, 'tagger'):
```

Comment:
Can you explain why those tests are needed?

Reply:
It prevents the secondary process from trying to notify a local tagger instance that doesn't exist.